components/form/form.module.css

```diff
@@ -0,0 +1,43 @@
+.input {
+  width: 100%;
+  padding: 1.5rem;
+  text-align: center;
+  transition: 0.5s;
+  background-size: 200% auto;
+  color: #574848;
```

Comment:
This could be a variable

Reply:
Yes, I'll have to update the css globals, thanks